docs/AppendixA-FAQ.md

```diff
@@ -1 +1,34 @@
+|                     |`The Candidate info validation has failed`|The upload scan contains at least one file that is NOT of type DICOM (.bmp or .pdf are common)|Remove any file in the upload that is not of type DICOM|
+|                     |`ERROR: This class is designed around the notion of a 'Study'. You can't use it with data from multiple studies.  The following study UIDs were found: '1.3.12.2.1107.5.2.43.66060.30000018010817270342400000004' '1.3.12.2.1107.5.2.43.66060.30000018011016312906600000007' The dicomTar execution has failed`|The upload contains acquisitions from two scanning sessions|Separate into two separate uploads|
+|                     |`Out of memory!`|The Transfer syntax of the uploaded scan is other than Little Endian Explicit (such as Implicit or JPEG lossless)|Use the DICOM toolkit `dcmconv` to convert. An example command that changes the transfer syntax to Little Endian Explicit on all DICOM files within a given folder is: `for i in $(find -type f); do dcmconv --write-xfer-little $i $i; done`|
+|                     |`Number of MINC files that will be considered for inserting into the database: 0. No data could be converted into valid MINC files. Localizers will not be considered!`|The upload contains only ‘localizer’ type SeriesDescription|Localizers are not processed by default in Loris|
+|                     |My upload DICOM has 6 modalities but only 5 are inserted; the 6th one has no MINC generated and nothing in the LORIS MRI Violated Scans module|Probably the scan is a that of a `localizer` acquisition. `tarchiveLoader` is specifically 'instructed' to exclude this modality|No action needed. This is the expected behavior of the LORIS-MRI insertion pipeline|
+|                     |My resting-state fMRI scans are tagged as task fMRI although I have 2 entries in the `mri_protocol` table|The resting-state scan has parameters that match those of the task entry of the `mri_protocol` table, and the task-related entry in the `mri_protocol` table precedes that of the resting-state fMRI|Make sure the `mri_protocol` table has parameters that discern between all the study acquired modalities in an **exclusive** manner (i.e. no two row entries have overlapping parameters across all their columns)|
+|                     |`no MINCs inserted`|Possibly all the MINC images are violated scans|Check the images headers details (in Imaging Browser or using `mincheader` against the `mri_protocol` table entries, and adjust the table protocol parameters accordingly|
```

Comment:
I guess it should be written: "Check the detail of the image headers" (got confirmed by Samir cause I was not sure myself either. ;)).

Reply:
thanks for going that extra mile! done (I wrote detailS of the image headers)